docs/en/8-MicroSim/GettingStarted.md

```diff
@@ -0,0 +1,125 @@
+
+# Getting started with Kubeflow Notebooks and OpenM++.
+
+This document is intended to act as an introduction to The AAWs Kubeflow Notebook servers and their use for the OpenM++ project.
+
+
+## Starting
+
+To access the AAW Kubeflow portal, navigate to the following website.
+
+[https://kubeflow.aaw.cloud.statcan.ca/](https://kubeflow.aaw.cloud.statcan.ca/) 
+
+This will redirect you to a Microsoft log-in page.
+
+![Login01](../images/Login01.png)
+
+Select the account you wish to use and proceed with the authentication.
```

Comment:
You may want to further clarify that the account must be the one that has been granted access to the namespace that they wish to access, which should be the statcan account for all internal users (cloud accounts are only supposed to be used by platform admins).